dataactvalidator/validation_handlers/validator.py

```diff
@@ -162,6 +162,7 @@ def cross_validate_sql(rules, submission_id, short_to_long_dict, first_file, sec
         failed_rows = conn.execute(
             rule.rule_sql.format(submission_id))
         if failed_rows.rowcount:
+            logger.info('VALIDATOR_INFO: Running query: %s on submission id %s', rule.query_name, submission_id)
```

Comment:
This line will only get logged if there are rows that failed. This needs to be logged before line 162. 

We can also remove the "VALIDATOR_INFO" part of the string. That was there originally to differentiate the logs from the broker but now we can do that in Kibana. I would also consider this format of logging:
`logger.info('Running %s on {submission_id: %s, job_id: %s}', rule.query_name,submission_id, job_id)`